### PR TITLE
Typo and grammar fixes in documentation

### PR DESCRIPTION
**Pull request by ghost**
This is a PR that originate in VSCode. However, because some affected files also reside on this repository, I created this PR.

This breaks no code, and the only behavior changed is the user-facing code, which was changed to be correct in grammar and spelling. There are also typos in the comments I corrected as well.

### CODE REVIEW

File: Extension/src/main.ts [hunk 1]
```diff
@@ -68,11 +68,11 @@ async function processRuntimeDependencies(): Promise<void> {
                 // Send the failure telemetry since postInstall will not be called.
                 sendTelemetry(await PlatformInformation.GetPlatformInformation());
             }
-        // The extension have been installed and activated before.
+            // The extension have been installed and activated before.
```
Comment (bobbrow) [inline Extension/src/main.ts]: Can you move this one inside the `else` block?  And fix `have` --> `has`?

File: Extension/src/main.ts [hunk 2]
```diff
@@ -68,11 +68,11 @@ async function processRuntimeDependencies(): Promise<void> {
...
         } else {
             await finalizeExtensionActivation();
         }
-    // No lock file, need to download and install dependencies.
+        // No lock file, need to download and install dependencies.
```
Comment (bobbrow) [inline Extension/src/main.ts]: Can you move this one inside the `else` block?

**Review by bobbrow — APPROVED** [review]
Thank you for the PR.

**Comment by ghost** [comment]
Fixed!

**Pull request merged**